wandb/cli/cli.py

```diff
@@ -1515,3 +1523,27 @@ def status(settings):
         click.echo(
             json.dumps(settings, sort_keys=True, indent=2, separators=(",", ": "))
         )
+
+
+@cli.command("disable", help="Disable W&B.")
+def disable():
+    api = InternalApi()
+    try:
+        api.set_setting("mode", "disabled", persist=True)
+        click.echo("W&B disabled.")
+    except configparser.Error:
+        click.echo(
+            "Unable to write config, copy and paste the following in your terminal to turn off W&B:\nexport WANDB_MODE=disabled"
+        )
+
+
+@cli.command("enable", help="Enable W&B.")
+def enable():
```

Comment:
@raubitsj  in the Spec we discussed potentially making this "disabled" and "enabled".  I think this reads better, but sacrifices consistency.  I'll let you make the final decision.

Reply:
i don't like the inconsistency. 

I think naming it "disabled"/"enabled" is awkward, but consistency is more important.
The preferred way to turn syncing on and off is now: offline/online (which are also adjectives)